spec.bs

```diff
@@ -3418,47 +3445,57 @@ algorithms to achieve the outcomes described in the above explanatory content.
 </div>
 
 <div algorithm>
-  Create a new algorithm, called <dfn>Derive a permissions policy directly from a fenced frame
-  config instance</dfn>.
+  Create a new algorithm, called <dfn>Create a permissions policy for a fenced navigable</dfn>.
 
-  Given null or an [=element=] (|container|), this algorithm returns a new [=permissions policy=].
+  Given an [=element=] (|container|) and an [=origin=] (|origin|), this algorithm returns a
```

Comment:
Do you know why this used to accept possibly-null elements, and why that is no longer necessary?

Reply:
An oversight. The one place where this algorithm is called checks that container is a fenced navigable container before passing it into this algorithm, so there's no chance that a null container will be passed into here.